cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -196,7 +196,7 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
 
     private fun downloadFromOrtResult(ortFile: File, failureMessages: MutableList<String>) {
         println(
-            "Downloading ${packageTypes.joinToString(" and ")} from ORT result file at " +
+            "Downloading ${packageTypes.joinToString(" and ") { "${it}s" }} from ORT result file at " +
```

Comment:
The curly braces around `it` are unnecessary.

Reply:
I don't think so, as you need to separate from the "s".

Reply:
OMG, you're right ^^